android-sdk/src/androidTest/java/com/mobilecoin/lib/MnemonicTest.java

```diff
@@ -44,41 +51,38 @@ public void create_mnemonic_12_words() throws BadEntropyException {
     public void create_mnemonic_24_words() throws BadEntropyException {
         String mnemonic = Mnemonics.bip39EntropyToMnemonic(Hex.toByteArray(
                 "6a36844131c70de644643935ac6610445d04adca6670f6bb6ee46935c873e214"));
-        assertEquals("health reflect aware glory ignore veteran " +
-                "bag mango cup glimpse lottery master " +
-                "space finger civil sock wall swarm " +
-                "ribbon sponsor frame delay marriage oyster", mnemonic);
+        assertEquals("health reflect aware glory ignore veteran bag mango cup glimpse lottery " +
+                "master space finger civil sock wall swarm ribbon sponsor frame delay marriage " +
+                "oyster", mnemonic);
```

Comment:
Consider moving the expected value string into a local variable. Might make it more readable / easier to update in the future.

Reply:
Let's find a solution to formatting strings and then refactor this. For now, I'll keep it like this.